Program.cs

```diff
@@ -12,6 +17,21 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                 .ConfigureLogging(builder =>
+                 {
+                     var appResourceBuilder = ResourceBuilder.CreateDefault()
+                         .AddService("fib-microsvc-log", "0.0.1");
+
+                     builder.AddOpenTelemetry(options =>
+                     {
+                         options.SetResourceBuilder(appResourceBuilder);
+                         options.AddOtlpExporter(option =>
+                         {
+                             option.Endpoint = new Uri("https://api.honeycomb.io");
+                             option.Headers = "x-honeycomb-team=xxxxx";
```

Comment:
This seems not ideal to me given I didn't know how to reuse the config from `appsettings.json` yet.